simple_approach_planner:
  ros__parameters:
    approach_velocity: 0.5
    approach_min_duration: 0.5

